src/main/java/com/twilio/sdk/resource/InstanceResource.java

```diff
@@ -153,6 +153,16 @@ public Integer getPropertyAsInteger(String name) {
 	}
 
 	/**
+	 * Gets the property as a Boolean value.
+	 *
+	 * @param name property name
+	 * @return Boolean value
+	 */
+	public Boolean getPropertyAsBoolean(String name) {
+		return Boolean.valueOf(getProperty(name));
```

Comment:
what i understand from your comments is, basically add another test case, that tests throwing exception.

Still, the steps you mentioned earlier are out of scope of the changes